client/db.go

```diff
@@ -208,6 +209,13 @@ type Store interface {
 
 	// ExecRequest executes the given GQL request against the [Store].
 	ExecRequest(context.Context, string) *RequestResult
+
+	// ACPModule returns the underlying acp module (optional). Incase the access control
+	// is turned off, then the returned acp module will have no value (will be empty).
+	ACPModule(context.Context) (immutable.Option[acp.ACPModule], error)
```

Comment:
todo: Please document these at least as well as the underlying calls in the `ACPModule`, if anything the documentation should be more comprehensive here, not the other way around.

Reply:
It is the public interface through which users will interact with it, users shouldn't have to dig through into an internal (acp) package to find out what it does.

Reply:
Ohhh, I have misread it.  I may have more comments in a little bit, but I'll resolve this thread :grin: